### PR TITLE
Double h1 tag on category page

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x
| Description?  | Double h1 tag when using filters. See https://github.com/PrestaShop/PrestaShop/issues/10470
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | #10470 
| How to test?  | Read ticket

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10470)

### CODE REVIEW

**Comment by PierreRambaud** [comment]
You can have many h1 per pages, depends on how to use section tags. I'm not in favor to merge this one.

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
@colinegin Can you confirm what I said in my comment with SEO team?

**Comment by MathiasReker** [comment]
Why would you want to have more than one h1 tag per page?

**Comment by PierreRambaud** [comment]
@Mreker If you use section, you should have split content by section (this is why it's call section ofc). Each section can have a specific title. And even google said it's ok to have multiple h1: https://www.youtube.com/watch?v=WsgrSxCmMbM

**Comment by MathiasReker** [comment]
Thanks for the YouTube video. I did not know about that.

I still think that` <p> `is better for this title. But I understand the video, so this PR can be closed if you think h1 is best.

**Comment by MathiasReker** [comment]
Maybe some search engine different from Google do not like more than one H1 tag per page.

**Comment by PierreRambaud** [comment]
@Mreker This is in the RFC https://tools.ietf.org/html/rfc1866#section-5.4 and google have at least 91% of the global search, and Bing 3.5% and I'm sure they are following the RFC ^^ (http://gs.statcounter.com/search-engine-market-share)

**Comment by MathiasReker** [comment]
Fine with me. Thanks.

**Comment by PierreRambaud** [comment]
One more article: http://www.suvaance.com/microblog/how-google-considers-multiple-h1-tags-in-one-page/

As I said in the issue ticket, don't forget we are in HTML5 now ;)

**Comment by PierreRambaud** [comment]
@Mreker You must edit this file instead: https://github.com/PrestaShop/PrestaShop/blob/develop/themes/classic/templates/catalog/_partials/category-header.tpl

**Comment by MathiasReker** [comment]
Okay, should I do anything. As I understand, it is not a bug?

**Comment by PierreRambaud** [comment]
The fix you did isn't a bug. But in `category-header.tpl` there is two h1 and only one in this section must be present. If you want to change your PR to make the change but I don't know how to test it, thinking one is for desktop and the other for mobile :/

**Comment by MathiasReker** [comment]
I will look into this. Thanks for clarifying :)

**Comment by PierreRambaud** [comment]
Thanks to you again for what you do =)

**Comment by MathiasReker** [comment]
You are welcome :)

**Comment by MathiasReker** [comment]
mobile
desktop

@PierreRambaud you are right, it depends on screensize. wdyt? not a bug, right?

**Comment by PierreRambaud** [comment]
Only one h1 must be set here. A refactor is needed :/

**Comment by MathiasReker** [comment]
What about this solution?

**Comment by MathiasReker** [comment]
I don't know where to place the css, this is just for demonstration.

**Comment by MathiasReker** [comment]
after
It looks like this now, with only one H1 tag.

The description was missing before, was that the attended behavior?

**Comment by PierreRambaud** [comment]
@CaptainYouz Your turn to check if everything is ok ;)
@colinegin Any idea about desired behavior?

File: themes/classic/assets/css/custom.css [hunk 1]
```diff
@@ -2,3 +2,22 @@
  * Custom code goes here.
  * A template should always ship with an empty custom.css
  */
+@media screen and (max-width: 767px) {
```
Comment (eternoendless) [inline themes/classic/assets/css/custom.css]: Theme css code should be set in the appropriate scss file (inside the `_dev` folder). This file is for shop customizations only, as stated in the comment above.

**Review by eternoendless — DISMISSED** [review]

File: themes/classic/assets/css/custom.css [hunk 2]
```diff
@@ -2,3 +2,22 @@
...
+	.category-cover{
+		display: none !important;
```
Comment (CaptainYouz) [inline themes/classic/assets/css/custom.css]: Why every style has `!important` declaration? 
If the style will affect a component globally, the style should be directly modified at the component level.
If the style is a custom one, it is better to create a specific class for that, so the style could be reused in the future (if needed).

File: themes/classic/assets/css/custom.css (hunk 1 above)
Comment (MathiasReker) [reply themes/classic/assets/css/custom.css]: @eternoendless I know the place was wrong :) I just didn't know where to place it. Thanks, I will update my PR

File: themes/classic/assets/css/custom.css (hunk 2 above)
Comment (MathiasReker) [reply themes/classic/assets/css/custom.css]: You are right, I will fix this.

**Comment by MathiasReker** [comment]
Mobile before:
mobile_before

Mobile after:
mobile_after

Desktop before:
desktop_after

Desktop after: 
desktop_before

I am not sure how to build assets, I tried to run `npm run build` in `_dev` but I get alot of errors.

File: themes/classic/assets/css/custom.css (hunk 1 above)
Comment (MathiasReker) [reply themes/classic/assets/css/custom.css]: Fixed, but I need help to build assets

File: themes/classic/assets/css/custom.css (hunk 2 above)
Comment (MathiasReker) [reply themes/classic/assets/css/custom.css]: Fixed

**Comment by colinegin** [comment]
Seems great to me ! What do you think @NadiaCamard and @TristanLDD ?
Thanks a lot for your work @Mreker :)

**Comment by TristanLDD** [comment]
That's ok for me, can you just use the font Noto Sans instead of arial for the text below the category name on mobile ?

**Comment by MathiasReker** [comment]
@TristanLDD Noto Sans is already default. In this case Noto Sans is overwritten by Arial, because the description has inline style. Please see attachment. 
inline style

I think this should be done in an other PR. But here is the related file to be changed: `install-dev\fixtures\fashion\langs\en\data\category.xml`

**Comment by MathiasReker** [comment]
@TristanLDD here you go: https://github.com/PrestaShop/PrestaShop/pull/10867 :-)

**Comment by MathiasReker** [comment]
Everything ok? :)

**Comment by TristanLDD** [comment]
Everything's ok for me ! thanks @Mreker

**Comment by CaptainYouz** [comment]
it's good for me too!

**Review by PierreRambaud — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
@Mreker Do you need help to build assets?

**Comment by MathiasReker** [comment]
@PierreRambaud yes, please. Can you please build it for me?

**Comment by MathiasReker** [comment]
@PierreRambaud Thank you. :)

**Comment by MathiasReker** [comment]
@PierreRambaud You built assets for js, not css?

**Comment by PierreRambaud** [comment]
@Mreker Done, and Yes I was building the wrong directory =)

**Review by Quetzacoalt91 — APPROVED** [review]
N/A on StarterTheme

**Comment by PierreRambaud** [comment]
Thanks @Mreker

**Pull request merged**